boostrap_test.go

```diff
@@ -8,12 +8,12 @@ import (
 )
 
 func TestDatadog(t *testing.T) {
-	ddCfg := config.DatadogConfig{}
+	ddCfg := new(config.DatadogConfig)
 
 	err := StartDatadog(ddCfg, ConnectionTypeHTTP)
```

Comment:
FYI, this test was broken not sure why in CI it's not executed anymore but `StartDatadog accepts the interface now, and then the structure must be passed by a pointer.

Reply:
Could be was not thinking about that actually